bitnami/influxdb/README.md

```diff
@@ -51,18 +91,14 @@ To make this process easier, the chart contains the `resourcesPreset` values, wh
 
 ### Update credentials
 
-Bitnami charts configure credentials at first boot. Any further change in the secrets or credentials require manual intervention. Follow these instructions:
+This chart supports creating a random admin token at first boot (not supported when using `memory` as object store) by setting both the `auth.enabled` and `createAdminTokenJob.enabled` parameters to `true`. Any further change in the credentials require manual intervention, pleaser refer instructions below:
```

Comment:
Add a note explaining how to use existingSecret, making it clear that it only works for an already initialized instance